src/tools/crawler/network.rs

```diff
@@ -23,6 +23,8 @@ pub struct KnownNode {
     pub protocol_version: Option<ProtocolVersion>,
     /// The node's user agent.
     pub user_agent: Option<VarStr>,
+    /// The node's best block height.
```

Comment:
the best block height? 😃

Reply:
Probably was thinking about something else 😃  Changed.